base/src/main/java/com/blazebit/expression/base/BaseContributor.java

```diff
@@ -107,6 +108,8 @@
     public static final String INTERVAL_TYPE_NAME = "Interval";
     public static final Class<?> STRING = String.class;
     public static final String STRING_TYPE_NAME = "String";
+    public static final String LOCAL_DATE_TYPE_NAME = "LocalDate";
+    public static final Class<?> LOCAL_DATE = LocalDate.class;
```

Comment:
I'd prefer if we use the name `Date` analogously to the use of `Time` even though we use `LocalTime` as java type.

```suggestion
    public static final String DATE_TYPE_NAME = "Date";
    public static final Class<?> DATE = LocalDate.class;
```